recipes/ipymidicontrols/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "ipymidicontrols" %}
+{% set version = "0.1.2" %}
+{% set hash = "35d9004e1a28b37c6a3158a132f36311e565f71b8046d44f47a0966539f44e99" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --ignore-installed --no-cache-dir"
```

No need for the arguments, they are all included by default now.

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
```

---

Thanks. I was wondering about that.